packages/writing-style/scripts/download-vale.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+const shelljs = require('shelljs');
+
+const version = '1.7.1';
+
+const binaryFileName = (() => {
+  switch (process.platform) {
+    case 'darwin':
+      return `vale_${version}_macOS_64-bit.tar.gz`;
+    case 'linux':
+      return `vale_${version}_Linux_64-bit.tar.gz`;
+    case 'window':
+      return `vale_${version}_Windows_64-bit.zip`;
+    default:
+      throw new Error(
+        `Cannot download vale binary, unsupported platform: ${process.platform}.`
+      );
+  }
+})();
+
+const archivePath = path.join(__dirname, `../${binaryFileName}`);
+const destPath = path.join(__dirname, '../bin');
+
+console.log('[writing-styles] Verifying vale installation...');
+if (fs.existsSync(path.join(destPath, 'vale'))) {
+  console.log(
+    '[writing-styles] Vale binary already installed, skipping installation...'
+  );
+} else {
+  shelljs.exec(
+    `curl -sLO https://github.com/errata-ai/vale/releases/download/v${version}/${binaryFileName}`
```

Comment:
This is a linux & mac os specific implementation, the curl command is not available OOTB on a windows installation.  Rather use node-fetch or direct node.js  HTTPS?

Reply:
Hmm good point. I’ll see if there is a cross platform package to do that, otherwise we can keep things as they are for now

Reply:
https://github.com/commercetools/commercetools-docs-kit/pull/131/commits/924649dfe205b85489a71b8b7a3c82cc020746e9